doc/account.md

```diff
@@ -0,0 +1,35 @@
+Account
+=======
+
+The `account` options in `config.json` allow you to set how long an account is viable at times during it's lifespan. There are current only settings for `initiate` and `complete`. We want the lifetime for an initial account creation to be short so we can clean up accounts which haven't been completed and won't have numerous files sitting in storage. Also we set a `complete` account to 6 months; to comply with PCI standards.
+
+The object assigned to each account section is passed into the OtDate object to set the expires for each lifetime.
+
+    "account": {
+        "completeLifetime": {
+            "months": 6
+        },
+        "initiateLifetime": {
+            "hours": 1
+        }
+    }
+
+
+Lifespan
+--------
+
+Account creation and use goes through several phases.
+
+Initiate
+--------
+
+The first part of an account's lifespan is `initiate`. This sets up a few pieces of data needed to properly generate and track an account. We generate an Account ID, MFA Key, abd Password Salt. An Email address must be passed in. These are then passed back to the user to store on their own and will be used to generate their password.
+
+This will save the account using a hash of the their `accountId`.
```

Comment:
That doesn't read well.  How about this?

This will save the account to a file named after the hash of their `accountId`.